syntax = "proto3";

package client_server.grpc.v1;

service CSService01 {
  rpc CSServiceMethod01 (stream CSService01Method01MessageIn01) returns (stream CSService01Method01MessageOut01) {}
}

message CSService01Method01MessageIn01 {
  string CSService01Method01MessageIn01Param01 = 1;
}

message CSService01Method01MessageOut01 {
  string CSService01Method01MessageOut01Param01 = 1;
}